cmd/spiffe-csi-driver/main.go

```diff
@@ -39,19 +40,32 @@ func main() {
 	}
 	log = zapr.NewLogger(zapLog)
 
+	// If the enforce-read-only-volume flag was not explicitly provided,
+	// instruct the user to set it. Either way, if it is set to false, or
+	// unset, emit a log educating users that the will be enforced in the
+	// future.
+	// TODO: enforce this by default in a future release.
+	if !isFlagSet("enforce-read-only-volume") {
+		log.Error(nil, "Pass the --enforce-read-only-volume flag to enforce that the CSI volume is marked read-only")
```

Comment:
Nit: Should this be logged at the warn level instead? Or is the error level intentional?

Reply:
For better or worse, logr has become somewhat of the defacto logger interface in k8s projects and `logr` doesn't believe in log levels. It's either informational (Info) or there is a problem (Error). There is a [philosophy](https://github.com/go-logr/logr#why-not-named-levels-like-infowarningerror) behind it.

Reply:
True, I forgot about that. 😉 Please ignore/resolve my two comments about this.

Reply:
I forgot too. The first thing I typed was `log.Warn` :laughing: and my compiler had to remind me.